installbuilder/datafiles/Base_DSC.data

```diff
@@ -325,6 +333,7 @@ done
 
 #if BUILD_OMS == 1
 chown -R omsagent /etc/opt/omi/conf/${{SHORT_NAME}}
+chown -R omsagent /opt/microsoft/omsagent/plugin
```

Comment:
Is this directory created as part of the omsagent package?  If so, then this is fine.  Otherwise, you might want to add a "mkdir -p /opt/microsoft/omsagent/plugin" before this